jxp/ch13ex2.pl

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/perl
+use strict;
+use warnings;
+use Cow;
+use Horse;
+use Sheep;
+use Mouse;
+use v5.10;
+
+say "Please enter a list of Animals";
+my $input = <STDIN>;
+
+my @animals = split " ",$input;
```

Comment:
Reading in `STDIN` into the one string to then split it up seems inefficient. You could build your `@animals` array like:

```perl
my @animals = do {
  local $/ = ' ';
  <STDIN>
};
```